lib/cli_script.dart

```diff
@@ -398,23 +397,25 @@ IOSink append(String path) => File(path).openWrite(mode: FileMode.append);
 ///
 /// If [name] is passed, it's used as the name of the spawned script.
 ///
+/// [Script.kill] returns `true` if the stream was interrupted and the script
+/// exits with [Script.exitCode] `143`, or `false` if the stream was already
+/// closed.
+///
+/// The [callback] can't be interrupted by calling [kill], but the [onSignal]
+/// callback allows capturing those signals so the callback may react
+/// appropriately. Calling [kill] returns `true` even without an [onSignal]
+/// callback because the stream is still interrupted.
```

Comment:
```suggestion
/// callback if the stream is interrupted.
```

This makes it clearer that killing the script after it's completed will return `false`.